### PR TITLE
[UserBundle] Make templates configurable

### PR DESCRIPTION
**Pull request by michalmarcinkowski**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | https://github.com/Sylius/Sylius/issues/3190
| License       | MIT

Make templates and forms configurable and separate UserBundle from Core.

### CODE REVIEW

File: src/Sylius/Bundle/UserBundle/Controller/SecurityController.php [hunk 1]
```diff
@@ -56,12 +59,13 @@ public function logoutAction(Request $request)
...
      * an extended controller to provide additional data for the login template.
      *
-     * @param array $data
+     * @param string $view The view name
+     * @param array  $data
      *
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    protected function renderLogin(array $data)
+    protected function renderLogin($view, array $data)
```
Comment (gperdomor) [inline src/Sylius/Bundle/UserBundle/Controller/SecurityController.php]: i think template is a better name :D

File: src/Sylius/Bundle/UserBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -1,9 +1,25 @@
 sylius:
-    user:
+    ui:
```
Comment (gperdomor) [inline src/Sylius/Bundle/UserBundle/Resources/translations/messages.en.yml]: why ui?

File: src/Sylius/Bundle/UserBundle/Controller/SecurityController.php (hunk 1 above)
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/UserBundle/Controller/SecurityController.php]: I keep to the Symfony naming http://api.symfony.com/2.0/Symfony/Bundle/FrameworkBundle/Controller/Controller.html#method_render

File: src/Sylius/Bundle/UserBundle/Resources/translations/messages.en.yml (hunk 2 above)
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/UserBundle/Resources/translations/messages.en.yml]: The translations keys will be all refactored and all UI related general phrases will be under `ui`, so we do not have to translate `id` 100+ times (for all entities). Discussed with @pjedrzejewski.

**Comment by michalmarcinkowski** [comment]
Closing in favour of #3197

**Pull request closed without merging**